lib/Conversion/LlvmToNeura/LlvmToNeuraPatterns.td

```diff
@@ -3,11 +3,17 @@ include "mlir/IR/PatternBase.td"
 include "mlir/Dialect/LLVMIR/LLVMOps.td"
 include "NeuraDialect/NeuraOps.td"
 
+// Floating point binary operations
 def : Pat<
   (LLVM_FAddOp $lhs, $rhs, $_fastmath),
   (Neura_FAddOp $lhs, $rhs)
 >;
 
+def : Pat<
+  (LLVM_FSubOp $lhs, $rhs, $_fastmath),
+  (Neura_FSubOp $lhs, $rhs)
+>;
+
```

Comment:
Yes. But FYI, we are moving such pattern conversion in the .cc: lib/Conversion/LlvmToNeura/LlvmToNeuraPass.cpp as shown in this PR: https://github.com/coredac/dataflow/pull/23/files#

The reason is we are making these op support additional operand like `predicate`.

But it is okay for you to add the conversion here. I can fix them in my PR.